computer_vision/object_detection/yolo_v4_tiny/run.py

```diff
@@ -85,6 +85,10 @@ def run_tf_fp32(model_path, batch_size, num_runs, timeout, images_path, anno_pat
 
 def main():
     args = parse_args()
+
+    if "COCO_IMG_PATH" or "COCO_ANNO_PATH" not in os.environ:
```

Comment:
I would move this logic to download_coco function so it doesn't have to be written each time.

Reply:
Also as images and annotations must correspond I would assume that if any of the two has been set manually the other one must have been forgotten and the intention of user should not be ignored and overridden.